sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/aggregate/PercentileSuite.scala

```diff
@@ -38,36 +38,43 @@ class PercentileSuite extends SparkFunSuite {
   }
 
```

Comment:
Seems we don't need to change this test suite at all. Now `PercentileDisc` is almost the same with `Percentile`, just a few flags are different. Testing `Percentile` should be good enough.

Reply:
OK. I will revert it.